libs/shared/geodb/src/lib/geodb.manager.ts

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Inject, Injectable } from '@nestjs/common';
+import { GeodbProvider } from './geodb.provider';
+import type { GeodbCityReader } from './geodb.provider';
+import { getLocationData } from './geodb.repository';
+import { GeodbManagerConfig } from './geodb.config';
+import { TaxAddress } from './geodb.types';
+
+Injectable();
+export class GeodbManager {
```

Comment:
Q: Is there a reason why you chose to spell it this way? Should it not be `GeoDBManager` (and all similar instances renamed to `GeoDB` or `geoDB` respectively). Looking at how it was written before (other fxa-* packages), I see references to GeoDB (or geoDB), and GeoDb (or geoDb).

It looks like the branded spelling is "GeoDB" so I would think even with camel or pascal case it would be capital **DB**.